discover/portal/articles/04-business-productivity/01-forms/05-success-page.markdown

```diff
@@ -1,22 +1,59 @@
 # Form Success Pages
 
-Once a user submits one of your whiz-bang forms, what next? Don't leave them
-feeling lost, wondering whether their form entry was truly submitted, and
-whether they should fill the form out again. Instead, configure a Success Page. 
-
-The success page is simple. It has a title in bold text, and a description beneath
-the title. If you want to redirect users to a different page in your site, use a
-[redirect page](/discover/portal/-/knowledge_base/7-0/creating-advanced-forms#redirecting-users-after-form-submission). The redirect page is a terminal page taht just gives form users a message that they've finished filling out the form and their submission was received successfully.
-
-So what should you put in the success page? Whatever you want, but you can try
+Once a user submits one of your whiz-bang forms, what next? How will they know
+they're done, and can close the browser window or tab? What if they think their
+submission didn't go through, and wonder if they need to fill out the whole form
+again? By default, submitting a form brings you back to the first page of the
+form, and it's now blank. Don't leave users feeling lost, relying on the default
+success message to assure them that their form entry was successfully submitted.
+Instead, configure a Success Page.
+
+The success page is simple. It has a title in bold text, and a description
+beneath the title. A common alternative to using a success page is to [redirect
+users to a different page in your site](/discover/portal/-/knowledge_base/7-0/creating-advanced-forms#redirecting-users-after-form-submission).
+
+The success page is a terminal page that just gives form users a message that
+they've finished filling out the form and their submission was received
+successfully. A success page can even urge users to close the browser window or
+tab.
+
+What should you put in the success page? Whatever you want, but you can try
 this if you are a serious individual with a serious form:
 [![Figure x: Here's a long, serious form success page.](../../../images/forms-success-page1.png)
 
 If you're writing a success page for a more "fun" form, like a registration form
 for a dog sled race, you can be more playful:
 [![Figure x: A potential dog sled race registration form's success page.](../../../images/forms-success-page.png)
 
-To configure a success page, click 
+To configure a success page, you must be in the form builder. Click on the form
+page's Action (![Actions](../../../images-dxp/icon-actions.png)) button and
+choose *Add Success Page*.
+
+![Figure x: Add a success page using the edit menu for the form page.](../../../images/forms-success-page-add.png)
+
+Once the success page is added to your form, fill in the Title and Description
+fields however you please. When the form is Saved and Published, the Success
```

Comment:
Sounds great for me! And sorry for didn't get it at first, but the fact I'm not a native speaker can lead me to not understand some expressions.